integration_testing/features/super-admin/super-admin-upgrade-notification.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Super admin receives upgrade notification
+    Super admin needs to be able to dismiss or click out of notification
+
+  Background:
+    Given I am signed in to Kolibri as super admin
+      And I see a notification message
```

Comment:
in order to test this, we still need a process that can be initiated

Do we have any ways of mocking the notification server or otherwise triggering fake notifications?

Reply:
In https://github.com/learningequality/kolibri/pull/4731, I mentioned doing something like `kolibri manage ping --once --server http://nutritionfacts-staging.nanoapp.io` from the command line. Is this ok for being included in a gherkin story?

Reply:
> Is this ok for being included in a gherkin story?

the only part I am unclear on is how the tester creates a message. Ideally, a manual tester would be able to generate actual messages on `nutritionfacts-staging` and have kolibri listen for and display them

Reply:
Could we generate a message locally via fixture or management command or in some other way? Or is this intended to also be testing the telemetry integration?

Reply:
If the latter, we could add an argument to the `ping` management command that adds a header/GET param to the request, which makes Nutritionfacts reply with a dummy message.

Reply:
There are two things I'd like to achieve:

1. manual testing of a few basic messages, essentially just to make sure the pathway from ping -> DB -> UI works for a few messages, version ranges, etc
2. ability to test delivery and preview a message before sending it out for real

Reply:
For 2, we could have a flag on nutritionfacts messages for filtering by "mode", so it would only send it back if the client had a particular mode set

Reply:
Might then be able to use that for 1 in some way, as well?

Reply:
that sounds like a great plan